reana/reana_benchmark/analyze.py

```diff
@@ -307,31 +295,35 @@ def _build_histogram_plot(
 
 def _build_total_time_histogram(
     df: pd.DataFrame, plot_parameters: Dict
-) -> (str, Figure):
+) -> Tuple[str, Figure]:
     title = plot_parameters["title"]
+    data = df[df["status"] == WorkflowStatus.finished]
+    series = data["runtime"] + data["pending_time"]
```

Comment:
Shouldn't we use `.dropna()` here as well?

Reply:
`finished` status always has `runtime` and `pending_time` but it can be confusing so I can add `dropna()`

Reply:
I decided to leave it as it is because `dropna()` can be confusing. Someone can assume that the "runtime" and "pending_time" columns could have different data (because `dropna` removed it). But if you have suggestions how to make it more clear maybe, I can do it.

Reply:
Yeah, I agree that it's better not to add it, since as you said `finished` workflows will always have `runtime` and `pending_time`, so there is no need for `dropna()`. But then we need to remove `dopna()` in the `_build_runtime_histogram()` (line 313)?

Reply:
Actually, `dropna()` does not seem very necessary, as `NaN` values are simply omitted from the plot. Or am I missing something?

Reply:
Haven't checked it, maybe. I would prefer to drop them just in case.